### PR TITLE
Remote Control Documentation 4.7 updates

### PR DESCRIPTION
**Pull request by bilal-alsharifi**

### CODE REVIEW

File: docs/Remote Control/index.md [hunk 1]
```diff
@@ -20,32 +20,62 @@ Currently, the Remote Control feature supports these modules:
 
 | Supported RC Modules |
 | ---------            |
-| Radio                |
 | Climate              |
+| Radio                |
+| Seat                 |
+| Audio                |
+| Light                |
+| HM Settings          |
```
Comment (BrettyWhite) [inline docs/Remote Control/index.md]:
```suggestion
| HMI Settings          |
```

File: docs/Remote Control/index.md [hunk 2]
```diff
@@ -20,32 +20,62 @@ Currently, the Remote Control feature supports these modules:
...
+| **Audio**       | Audio volume | 0%-100%| Get/Set/Notification | The audio source volume level |
+|                 | Audio Source | MOBILE_APP, RADIO_TUNER, CD, BLUETOOTH, USB, etc. see PrimaryAudioSource| Get/Set/Notification | defines one of the available audio sources |
+|                 | keep Context | true, false| Set only | control whether HMI shall keep current application context or switch to default media UI/APP associated with the audio source|
+|                 | Equilizer Settings | Struct {Channel ID as integer, Channel setting as 0%-100%} | Get/Set/Notification | Defines the list of supported channels (band) and their current/desired settings on HMI
+| **Light**       | Light Status | ON, OFF| Get/Set/Notification | turn on/off a single light or all lights in a group |
+|                 | Light Density | float 0.0-1.0| Get/Set/Notification | change the density/dim a single light or all lights in a group|
+|                 | Light Color | RGB color| Get/Set/Notification | change the color scheme of a single light or all lights in a group|
+| **HM Settings** | Display Mode | DAY, NIGHT, AUTO | Get/Set/Notification | Current display mode of the HMI display |
```
Comment (BrettyWhite) [inline docs/Remote Control/index.md]:
```suggestion
| **HMI Settings** | Display Mode | DAY, NIGHT, AUTO | Get/Set/Notification | Current display mode of the HMI display |
```

File: docs/Remote Control/index.md [hunk 3]
```diff
@@ -164,37 +201,31 @@ Another unique feature of Remote Control is the ability to send simulated button
...
+}
 ```
 #### Subscribing to changes
 
 It is also possible to subscribe to changes in data associated with supported modules.
 
 To do so, during your `GET` request for data, simply add in `setSubscribe(Boolean)`. To unsubscribe, send the request again with the boolean set to `False`. A code sample for setting the subscription is in the `GET` example above.
 
-The response to a subscription will come in a form of a notification. You can receive this notification through the `IProxyListenerALM` that was supplied to the `SdlProxyALM` object; the method `onOnInteriorVehicleData` will be called when the RPC is received.
+The response to a subscription will come in a form of a notification. You can receive this notification by adding  a notification listener for `OnInteriorVehicleData`.
```
Comment (BrettyWhite) [inline docs/Remote Control/index.md]:
```suggestion
The response to a subscription will come in a form of a notification. You can receive this notification by adding a notification listener for `OnInteriorVehicleData`.
```

File: docs/Remote Control/index.md [hunk 4]
```diff
@@ -54,28 +84,28 @@ The system shall list all available buttons for Remote Control in the `RemoteCon
...
+|             | EJECT |
+|             | SOURCE |
+|             | SHUFFLE |
+|             | REPEAT |
 
 ## Integration
 
 !!! NOTE
-For Remote Control to work, the head unit must support SDL Core Version 4.4 or newer
+For Remote Control to work, the head unit must support SDL Core Version 4.4 or newer. Also the appHMIType should be set to `REMOTE_CONTROL`.
```
Comment (joeygrover) [inline docs/Remote Control/index.md]:
```suggestion
For Remote Control to work, the head unit must support SDL Core Version 4.4 or newer. Also your app's appHMIType should be set to `REMOTE_CONTROL`.
```

File: docs/Remote Control/index.md [hunk 5]
```diff
@@ -164,37 +201,31 @@ Another unique feature of Remote Control is the ability to send simulated button
...
+}
 ```
 #### Subscribing to changes
 
 It is also possible to subscribe to changes in data associated with supported modules.
 
 To do so, during your `GET` request for data, simply add in `setSubscribe(Boolean)`. To unsubscribe, send the request again with the boolean set to `False`. A code sample for setting the subscription is in the `GET` example above.
 
-The response to a subscription will come in a form of a notification. You can receive this notification through the `IProxyListenerALM` that was supplied to the `SdlProxyALM` object; the method `onOnInteriorVehicleData` will be called when the RPC is received.
+The response to a subscription will come in a form of a notification. You can receive this notification by adding a notification listener for `OnInteriorVehicleData`.
```
Comment (joeygrover) [inline docs/Remote Control/index.md]: There should be a note saying that adding this listener should happen before sending the `GetInteriorVehicleData` request.

**Pull request merged**